src/sentry/tasks/summaries/weekly_reports.py

```diff
@@ -145,6 +145,11 @@ def prepare_organization_report(
             },
         )
         return
+    target_user_id: int | None = None
+    if isinstance(target_user, User):
```

Comment:
```suggestion
    if isinstance(target_user, (User, RpcUser)):
```

I think the callers for this in getsentry are passing in RpcUser not user, and the type hint here is lies.